channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -148,7 +148,7 @@ function _createEnvelope<T>(logger: IDiagnosticLogger, envelopeType: string, tel
     _setValueIf(envelope, "sampleRate", telemetryItem[SampleRate]);
     if ((telemetryItem[strBaseData] || {}).startTime) {
         envelope.time = toISOString(telemetryItem[strBaseData].startTime);
-    }
+    } // siyu 3.0.3 time = loadEvent time, use startTime to record it
```

Comment:
This comment is specifically for the page view event, however, this code runs for every event and is the mechanism used for the dependencies (ajax) requests.

So qualifying that since v3.0.3 the pageview event ..(etc).. would be better.